pkg/action/executor/quota_util.go

```diff
@@ -307,3 +310,26 @@ func getCodecForGV(gv schema.GroupVersion) (runtime.Codec, error) {
 	codec := codecs.CodecForVersions(serializerInfo.Serializer, codecs.UniversalDeserializer(), gv, nil)
 	return codec, nil
 }
+
+func addGCLabelOnQuota(quota *corev1.ResourceQuota) {
+	labels := quota.GetLabels()
+	if labels == nil {
+		labels = make(map[string]string)
+	}
+	labels[TurboGCLabelKey] = TurboGCLabelVal
+
+	quota.SetLabels(labels)
+}
+
+func RemoveGCLabelFromQuota(quota *corev1.ResourceQuota) {
+	labels := quota.GetLabels()
+	if labels == nil {
+		// nothing to do
+		return
+	}
+	if _, exists := labels[TurboGCLabelKey]; exists {
+		delete(labels, TurboGCLabelKey)
+	}
+
+	quota.SetLabels(labels)
```

Comment:
nit: Move this into the `exists` block after `delete(labels, TurboGCLabelKey)`

Reply:
Done!